### DISCUSSION TITLE
onedrive api

### DISCUSSION THREAD

**Discussion by focusv** [cguZZman/plugin.onedrive#59] [Ideas]
Is there anyway to use own Client ID and Key to avoid throttling? For reference: https://rclone.org/onedrive/#getting-your-own-client-id-and-key

**Comment by cguZZman** [comment]
Yes. You need to run your own drive-login server: https://github.com/cguZZman/drive-login
There is no documentation now, but you can start installing it and documenting the process and share it here.

**Comment by little-fish** [comment]
Hi @cguZZman.
I am trying to host my own drive-login but got stuck at MS side I guess. During logging process, after I specify my MS account's email and accept access to the application, I immediately get 401. Can you please share your MS Application's manifest (obfuscated of course). There has to be something I didn't set.
Thank you.
